framework/contracts/native/ibc-host/tests/integration.rs

```diff
@@ -104,48 +106,104 @@ fn account_creation() -> anyhow::Result<()> {
     Ok(())
 }
 
+#[test]
+fn cannot_register_proxy_as_non_owner() -> anyhow::Result<()> {
+    let sender = Addr::unchecked("sender");
+    let chain = Mock::new(&sender);
+
+    let admin = Addr::unchecked("admin");
```

Comment:
We have a bunch of consts defined for addresses, would be nice if you could use them in your tests: https://github.com/AbstractSDK/abstract/blob/31ff9b08d7400fcbb3c021a24cecb49446edb30c/framework/packages/abstract-testing/src/lib.rs#L15